internal/controller/components/workbenches/workbenches.go

```diff
@@ -55,6 +55,7 @@ func (s *componentHandler) Init(platform common.Platform) error {
 	if err := odhdeploy.ApplyParams(nbcManifestInfo.String(), "params.env", map[string]string{
 		"odh-notebook-controller-image": "RELATED_IMAGE_ODH_NOTEBOOK_CONTROLLER_IMAGE",
 		"oauth-proxy-image":             "RELATED_IMAGE_OSE_OAUTH_PROXY_IMAGE",
```

Comment:
I wonder whether we should keep this record here? Just to be safe? :thinking:

Reply:
this image probably wont be in use after 2.25